src/lib/components/Molecule3dViewer.js

```diff
@@ -25,6 +25,16 @@ Molecule3dViewer.defaultProps = {
     selectionType: 'atom',
     backgroundColor: '#FFFFFF',
     backgroundOpacity: 0,
+    zoom: {
+        factor: 0.8,
+        animationDuration: 0,
+        fixedPath: true,
+    },
+    zoomTo: {
+        sel: {},
+        animationDuration: 0,
+        fixedPath: true,
+    },
```

Comment:
Nested values behave oddly in `defaultProps` - if you just provide `zoom: {factor: 2}` you'll lose `animationDuration: 0` and `fixedPath: true`. That's fine for `animationDuration` because it seems like its default is in fact `0`, but `fixedPath` looks like it defaults to `false`? If I'm understanding it correctly though, maybe we'd actually be better off with `false`? The [docs](http://3dmol.csb.pitt.edu/doc/$3Dmol.GLViewer.html#zoom) say:

> if true animation is constrained to requested motion, overriding updates that happen during the animation

In the `zoomTo` / `zoom` chain we have, is `zoom` an "update that happens during the animation" of `zoomTo`? If I wanted a button that animates to a particular view, from whatever the user had focused on, would this make a smooth single 2-second animation to that view?
```js
zoomTo: {sel: importantThing, animationDuration: 2000, fixedPath: false}
```
(noting that `onRenderNewData` will call `glviewer.zoom` after `glviewer.zoomTo` no matter what, but I'm happy with the default 0.8 zoom as an endpoint)

Reply:
The main difference in practice is that `fixedPath` forcibly maintains the view as it zooms in - rotating the view snaps it back to the original view it started at while zooming in. I agree that it makes sense to set it to `false` by default, it's less intrusive and more smooth. This does mean that you couple the two together - the animation view begins from whatever you set the `zoom` to rather than some predefined default. 

As far as chaining animations, that is correct, it would make a single 2-second animation which begins from and maintains the specified `zoom` factor to the `sel` residue. 

 On a side note, there's pretty limited documentation on the `zoomTo` selection syntax, but adapting over [the 3dMol.js tests](https://github.com/3dmol/3Dmol.js/blob/bb653fa0cc9d940b78d5c61d7a7390ff3dab471c/tests/auto/tests/test98.js) worked out! I've added additional `PropType` constraints which should help make this a bit clearer. Here's the snippet for a small demo.
 
 I've made the changes in c82d1a9.
 
 ```
 import json
import six.moves.urllib.request as urlreq
from six import PY3

import dash
import dash_bio as dashbio
import dash_html_components as html
import dash_core_components as dcc

app = dash.Dash(__name__)

model_data = urlreq.urlopen(
    'https://raw.githubusercontent.com/plotly/dash-bio-docs-files/master/' +
    'mol3d/model_data.js'
).read()
styles_data = urlreq.urlopen(
    'https://raw.githubusercontent.com/plotly/dash-bio-docs-files/master/' +
    'mol3d/styles_data.js'
).read()

if PY3:
    model_data = model_data.decode('utf-8')
    styles_data = styles_data.decode('utf-8')

model_data = json.loads(model_data)
styles_data = json.loads(styles_data)

app.layout = html.Div([
    dashbio.Molecule3dViewer(
        id='my-dashbio-molecule3d',
        styles=styles_data,
        modelData=model_data,
        zoomTo={
            "sel": {"chain":'X',"resi":115},
            "animationDuration": 2000,
            "fixedPath": False
        },
        zoom={"factor": 0.6}
    ),
])




if __name__ == '__main__':
    app.run_server(debug=True)
```